app/services/analytics_events.rb

```diff
@@ -3162,6 +3162,34 @@ def idv_request_letter_visited(
     )
   end
 
+  # Acuant SDK errored after loading but before initialization
+  # @param [Boolean] success
+  # @param [String] error_message
+  # @param [Boolean] liveness_checking_required Whether or not the selfie is required
+  # @param [String] acuant_version
+  # @param [Integer] captureAttempts number of attempts to capture / upload an image
+  #                  (previously called "attempt")
```

Comment:
The goal was to make it more readable and clear that this line is a continuation of the line above it, rather than another param. I noticed this is in use elsewhere in the file.

I'm not tied to it though if you think it is actually less clear this way!

Reply:
Thanks for the explanation.  The current spacing is fine by me. 👍🏻